public/css/main.css

```
@@ -14,3 +14,7 @@ h3 {
 .row .col.s4 {
   height: 17em !important;
 }
+
+.repoForks{
```

Due to linting errors.
```suggestion
.repoForks {
```